spec/controllers/social_networking/comments_controller_spec.rb

```diff
@@ -22,6 +22,7 @@ module SocialNetworking
     end
     let(:comment) do
       double("comment",
```

Comment:
I know this is assigned to @eschlange, but I have one thought here: should this be an `instance_double` instead?